src/main/java/org/snomed/snowstorm/core/data/services/ConceptUpdateHelper.java

```diff
@@ -265,6 +265,11 @@ PersistedComponents saveNewOrUpdatedConcepts(
 			// Axiom refset members
 			markDeletionsAndUpdates(newVersionConcept, existingConcept, existingConceptFromParent, Concept::getAllOwlAxiomMembers,
 					defaultModuleId, refsetMembersToPersist, rebaseConflictSave);
+			
+			// Annotation refset members
+			markDeletionsAndUpdates(newVersionConcept, existingConcept, existingConceptFromParent, Concept::getAllAnnotationMembers,
```

Comment:
@QuyenLy87 when deleting a Concept with an annotation, the Concept is deleted but the annotation isn't. Is this expected?

Reply:
Hi @dmcgihtsdo, Good catch. I've updated the logic. You can find the test case here
https://github.com/IHTSDO/snowstorm/blob/18db3d6c2bc27efcbc9ad71f38fb26a2c48f41f0/src/test/java/org/snomed/snowstorm/core/data/services/ConceptServiceTest.java#L236